File: CursoPython3/exer/lista_64_Exer/exer23.py
"""
23) Escreva um programa que exiba as seguintes opções e realize os que se pede em cada uma delas: 
1 – Adição 
2 – Subtração 
3 – Multiplicação 
4 – Divisão 
"""

option = 0

while (option != 5):
    print("Escolha uma opção:")
    print("1 - Adição")
    print("2 - Subtração")
    print("3 - Multiplicação")
    print("4 - Divisão")
    print("5 - Sair")

    option = int(input("Op: "))

    print("Informe o primeiro número:")
    num1 = int(input())
    print("Informe o segundo número:")
    num2 = int(input())

    if (option == 1):
        addition = num1 + num2
        print(f"A soma entre {num1} e {num2} é igual a {addition}")
        break
    elif (option == 2):
        subtraction = num1 - num2
        print(f"A subtração entre {num1} e {num2} é igual a {subtraction}")
        break
    elif (option == 3):
        multiplication = num1 * num2
        print(
            f"A multiplicação entre {num1} e {num2} é igual a {multiplication}")
        break
    elif (option == 4):
        division = num1 / num2
        print(f"A divisão entre {num1} e {num2} é igual a {division}")
        break
    elif (option == 5):
        print("Saindo...")
        break
    else:
        print("Opção inválida!")

File: Curso de Java/Exer/src/lista64Exer/Exer23.java

/*******************************************************************************
 * 23) Escreva um programa que exiba as seguintes opções e realize os que se pede em cada
 * uma delas:
 *		1 – Adição
 *		2 – Subtração
 *		3 – Multiplicação
 *		4 – Divisão
 *******************************************************************************/

import java.util.Locale;
import java.util.Scanner;

public class Exer23 {

    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int option = 0;
        double result = 0;

        do {
            System.out.println("Escolha uma opção:");
            System.out.println("1 - Adição");
            System.out.println("2 - Subtração");
            System.out.println("3 - Multiplicação");
            System.out.println("4 - Divisão");
            System.out.println("0 - Sair");
            System.out.print("Opção escolhida: ");

            option = sc.nextInt();

            switch (option) {
                case 1:
                    System.out.print("Digite o primeiro número: ");
                    double number1 = sc.nextDouble();
                    System.out.print("Digite o segundo número: ");
                    double number2 = sc.nextDouble();
                    result = number1 + number2;
                    System.out.println("Resultado: " + result);
                    break;
                case 2:
                    System.out.print("Digite o primeiro número: ");
                    number1 = sc.nextDouble();
                    System.out.print("Digite o segundo número: ");
                    number2 = sc.nextDouble();
                    result = number1 - number2;
                    System.out.println("Resultado: " + result);
                    break;
                case 3:
                    System.out.print("Digite o primeiro número: ");
                    number1 = sc.nextDouble();
                    System.out.print("Digite o segundo número: ");
                    number2 = sc.nextDouble();
                    result = number1 * number2;
                    System.out.println("Resultado: " + result);
                    break;
                case 4:
                    System.out.print("Digite o primeiro número: ");
                    number1 = sc.nextDouble();
                    System.out.print("Digite o segundo número: ");
                    number2 = sc.nextDouble();
                    if (number2 == 0) {
                        System.out.println("Não é possível dividir por zero.");
                    } else {
                        result = number1 / number2;
                        System.out.println("Resultado: " + result);
                    }
                    break;
                case 0:
                    System.out.println("Saindo...");
                    break;
                default:
                    System.out.println("Opção inválida.");
                    break;
            }
        } while (option != 0);

        sc.close();
    }

}
